networkmanager/traffic.go

```diff
@@ -68,24 +69,31 @@ type trafficData struct {
 }
 
 type trafficMonitoring struct {
-	iptables         *iptables.IPTables
-	trafficPeriod    int
-	skipAddresses    string
-	inChain          string
-	outChain         string
-	trafficMap       map[string]*trafficData
-	serviceChainsMap map[string]*trafficChains
-	trafficStorage   TrafficStorage
+	iptables          *iptables.IPTables
+	trafficPeriod     int
+	skipAddresses     string
+	inChain           string
+	outChain          string
+	trafficMap        map[string]*trafficData
+	instanceChainsMap map[string]*trafficChains
+	trafficStorage    TrafficStorage
 }
 
+/***********************************************************************************************************************
+ * Vars
+ **********************************************************************************************************************/
+
+// ErrNotExist not exist entry error.
+var ErrNotExist = errors.New("entry does not exist")
+
 func newTrafficMonitor(trafficStorage TrafficStorage) (monitor *trafficMonitoring, err error) {
```

Comment:
Could you move newTrafficMonitor function under private section?

Reply:
Done